common/djangoapps/student/tests/test_certificates.py

```diff
@@ -111,6 +112,14 @@ def test_linked_student_to_web_view_credential(self, enrollment_mode):
         self.assertContains(response, u'View Test_Certificate')
         self.assertContains(response, test_url)
 
+    @ddt.data('verified', 'honor', 'professional')
+    def test_unverified_certificate_message(self, enrollment_mode):
+        cert = self._create_certificate(enrollment_mode)
+        cert.status = CertificateStatuses.unverified
+        cert.save()
+        response = self.client.get(reverse('dashboard'))
+        self.assertContains(response, u'do not have a current verified identity with edX')
```

Comment:
Instead of hardcoding edX, use the platform name from settings so these tests don't fail if the platform name is changed.